pxd.c

```diff
@@ -1132,7 +1132,7 @@ int pxd_init(void)
 out_misc:
 	misc_deregister(&pxd_miscdev);
 out_fuse:
-	for (j = 0; j < i; ++j) {
+	for (j = 0; j <= i; ++j) {
```

Comment:
I could, but isn't this simpler? This also keeps the failure handling logic in one place (the error labels) rather than sprinkling them all over the function.